### PR TITLE
Added Electronic drum kit Project

### PR DESCRIPTION
**Pull request by JomanaMahmoud**
Closes: #92 

- **Title:** Electronic Drum Kit
- **Name:** Jomana Tantawy
- **Idenitfy yourself:** Contributor in GSSOC-EXT and Hacktoberfest

### Describe the add-ons or changes you've made 📃
In this update, I added a new **Electric Drum Kit** feature to the project under the `vanilla-js` folder. This project allows users to create drum beats by clicking on buttons or pressing corresponding keys on the keyboard, which trigger sounds for different drum components. Additionally, I've ensured that the interactive drum keys display a hover effect when pressed. This fun and dynamic project adds an exciting element to the website, allowing for both mouse and keyboard interaction. The implementation enhances user experience and brings more interactive functionality to the project.

### Screenshots 📷
screenshot

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: Vanilla-JS-Projects/Basic/Electronic-Drum-Kit/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<h1 align='center'><b>💥 Electronic Drum Kit 💥</b></h1>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Tech Stack Used 🎮</h3>
+<ul align="left">
+    <li><img src="https://img.shields.io/badge/HTML5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white" alt="HTML5 Badge"></li>
+    <li><img src="https://img.shields.io/badge/CSS3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white" alt="CSS3 Badge"></li>
+    <li><img src="https://img.shields.io/badge/JavaScript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E" alt="JavaScript Badge"></li>
+</ul>
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/Electronic-Drum-Kit/README.md:10]: Don't create this list.
image

You have to un-comment those tech stacks which are necessary for this project

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Vanilla-JS-Projects/Basic/Electronic-Drum-Kit/README.md (hunk 1 above)
Comment (JomanaMahmoud) [reply Vanilla-JS-Projects/Basic/Electronic-Drum-Kit/README.md:10]: Do you mean just remove the pictures and add them as text?

Comment (Avdhesh-Varshney) [reply Vanilla-JS-Projects/Basic/Electronic-Drum-Kit/README.md:10]: No, in readme there are some of tech stacks links are shown which are commentted. You have to remove those tech stack comments which are necessary to show for your project. As here only HTML, CSS and JS

Comment (JomanaMahmoud) [reply Vanilla-JS-Projects/Basic/Electronic-Drum-Kit/README.md]: done check it out

**Review by Avdhesh-Varshney — APPROVED** [review]

**Pull request merged**